portfolio/index.html

```diff
@@ -0,0 +1,137 @@
+<link href="https://fonts.googleapis.com/css?family=Syncopate" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300" rel="stylesheet">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+
+<div class="container-fluid">
```

Comment:
I think you should get rid of this wrapper div here. as far as I understand bootstrap, containers are only useful for wrapping grid rows which you only use for your portfolio section. And because you wrap the entire page in this div, you have to override the padding in your css so your background images cover the page.